src/management/projects.ts

```diff
@@ -12,14 +12,14 @@ export interface FirebaseProjectMetadata {
   projectId: string;
   projectNumber: string;
   displayName: string;
-  resources: DefaultProjectResources;
+  resources?: DefaultProjectResources;
 }
 
 export interface DefaultProjectResources {
```

Comment:
Are _all_ these properties really optional now? Why that change?

Reply:
it turns out that these resources may not exist in a Firebase project, for example with deferred location a new project can be created without specifying `locationId`.

Reply:
huh. the more you know.

Reply:
Confirming this is true. Due to deferred project resource creation, all should be optional now.